lighthouse-treemap/types/treemap.d.ts

```diff
@@ -0,0 +1,37 @@
+// import _Util2 = require('webtreemap-cdt'); // TODO: types.
+import _Util = require('../app/src/util.js');
+import {RootNodeContainer as _RootNodeContainer} from '../../lighthouse-core/audits/script-treemap-data';
+import {Node as _Node} from '../../lighthouse-core/audits/script-treemap-data';
+import '../../types/lhr';
+import '../../types/audit-details';
+
+declare global {
+  module Treemap {
+    interface Options {
```

Comment:
will this eventually have options in it too?

right now options feels like a bit of a misnomer, `payload`, `initialization data`, do any of those seem right?

I'm mostly pushing on this a bit because it feels like we are about to get some UI state for this app, which feels distinctly different from an initial payload. Maybe it's the intention to have all UI state configurable from the initial payload though?